qa/sc_evm_rpc_web3_sha3.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+from SidechainTestFramework.account.ac_chain_setup import AccountChainSetup
+from test_framework.util import assert_equal, assert_false, assert_true
+
+
+class SCEvmRpcWeb3Sha3(AccountChainSetup):
```

Comment:
This test should be added in run_sc_tests.sh

Reply:
As you've suggested, let's put the checks as a part of `sc_evm_rpc_web3_methods`.

Reply:
Done
Integration test moved to `sc_evm_rpc_web3_methods.py` and deleted `sc_evm_rpc_web3_sha3_py`